tools/docker/builder/openwrt/profiles_feeds/netgear-rax40.yml

```diff
@@ -0,0 +1,30 @@
+  - name: feed_target_mips
+    uri: https://intel.prpl.dev/prplmesh/feed_target_mips.git
+    hash: 78b0b2e9675ced7dd57829da7a27592dbb483fd3
+  - name: feed_datapath
+    uri: https://intel.prpl.dev/intel/feed_datapath.git
+    hash: 23afe7e659ed6717224659aef74af2bb11d8091a
+  - name: feed_ppa
+    uri: https://intel.prpl.dev/intel/feed_ppa.git
+    hash: 4f49fa34bbd51c7dd218df164e32c17694bed421
+  - name: feed_gphy_firmware
+    uri: https://intel.prpl.dev/intel/feed_gphy_firmware.git
+    hash: cbf75f2abde587ac2c5462d117b98e3dadaae5b6
+  - name: feed_switch_api
+    uri: https://intel.prpl.dev/intel/feed_switch_api.git
+    hash: af078af5408654ca2e759ea7afcf92163282e86e
+  - name: feed_wlan_6x
+    uri: https://intel.prpl.dev/prplmesh/feed_wlan_6x.git
+    hash: 8b7ff7d902f35b41864fbf306d45ad4ed91eaa89
+  - name: feed_prpl
+    uri: https://git.prpl.dev/prplmesh/feed-prpl.git
```

Comment:
Can we get these changes upstream so we can use the upstream one instead?

Reply:
I used the prplmesh forks first so I don't rely on the changes to be accepted upstream, but by know I think they have all been accepted.
I'll first merge this PR as is, and since we anyway need a follow-up PR to do some cleanup, it will be done there.